cmd/cape/cmd/token.go

```diff
@@ -226,4 +254,41 @@ func generateKeyPair() error {
 	return nil
 }
 
+func doGet(functionID string, url string, insecure bool, auth entities.FunctionAuth) error {
+	endpoint := fmt.Sprintf("%s/v1/functions/%s", url, functionID)
+
+	req, err := http.NewRequest("GET", endpoint, nil)
+	if err != nil {
+		return err
+	}
+
+	var bearer = "Bearer " + auth.Token
+	req.Header.Add("Authorization", bearer)
+
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return UserError{Msg: "failed to verify function ownership", Err: fmt.Errorf("cannot complete http request: %s", err)}
+	}
+
+	switch res.StatusCode {
+	case http.StatusNotFound:
+		return UserError{Msg: "function not found"}
+	case http.StatusUnauthorized:
+		return UserError{Msg: "unauthorized to create a function token for function"}
+	case http.StatusOK:
+
+	default:
+		return UserError{Msg: fmt.Sprintf("expected 200, got server response code %d", res.StatusCode)}
```

Comment:
Only the not found & unauthorized would be user errors in my opinion

Reply:
Got it.